src/routes/(split)/components/page-header/+page.md

```diff
@@ -0,0 +1,137 @@
+<script lang="ts">
+    import PageHeader from "$lib/components/PageHeader.svelte";
+    import Card from "$lib/components/Card.svelte";
+    import icpLogo from "$lib/assets/icp-rounded.svg";
+
+</script>
+
+# Page Header
+
+Used for settings and detail pages to have a first overview of the entity.
+
+```html
+<PageHeader>
+  <span slot="topLeft">Top left</span>
+  <span slot="topRight">Top right</span>
+  <h1 slot="title">Title</h1>
+  <h3 slot="subTitle">This page header is inside a card</h3>
+</PageHeader>
+```
+
+## Properties
+
+| Property | Description                                                     | Type                    | Default     |
+| -------- | --------------------------------------------------------------- | ----------------------- | ----------- |
+| `testId` | Add a `data-tid` attribute to the DOM, useful for test purpose. | `string` or `undefined` | `undefined` |
+
+## Slots
+
+| Slot name  | Description                                        |
+| ---------- | -------------------------------------------------- |
+| `title`    | The main information to render.                    |
```

Comment:
I know I'm being picky here so sorry in advance for the question but, if it's an "information", why not call the slot "information"? 

Maybe use the word "title" for the description of the slot as well. Same for subtitle

Reply:
I was looking for synonyms of the name of the prop. Let me rephrase it.